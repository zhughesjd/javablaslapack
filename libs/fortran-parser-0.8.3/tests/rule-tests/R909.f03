
! Testing close spec list.
close(10)
close(20, status='keep',err='foo')
end
