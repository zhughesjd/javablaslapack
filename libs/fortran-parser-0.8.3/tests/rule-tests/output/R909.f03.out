1101:begin
909:list-begin
406 10 null
909:list 1
909:list-begin
406 20 null
428 null null 'keep'
428 null null 'foo'
909:list 3
1103
1101 false true false
